gradebookng/tool/src/java/org/sakaiproject/gradebookng/tool/panels/StudentCompareGradesPanel.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html xmlns:wicket="http://wicket.apache.org">
+
+<body>
+    <wicket:panel>
+        <div id="studentGradesComparison-ModalContent">
+            <div class="studentGradesComparison-Header">
+                <span>
+                    <wicket:message key="comparegrades.modal.subtitle.label" />
+                </span>
+                <span wicket:id="gradeItemLabel">
+                    Item name
+                </span>
+            </div>
+            <div class="studentGradesComparison-TableContainer">
+                <table class="table table-bordered table-condensed table-hover">
+                    <thead>
+                        <th wicket:id="studentNameHeaderLabel">
+                            Student Name
+                        </th>
+                        <th wicket:id="teacherCommentHeaderLabel">
+                            Teacher comments
+                        </th>
+                        <th wicket:id="gradeHeaderLabel">
+                            Grade
```

Comment:
Hardcoded

Reply:
This is a wicket tag, and the hard coded value here will be replaced by the wicket message specified in Java. This is a non-issue.